### ISSUE TITLE
Cannot accurately identify wiki paths with Chinese characters

### ISSUE THREAD

**Issue by VimWei** [lervag/wiki.vim#394]
# Description

When following a wiki link with Chinese characters, wiki.vim is unable to recognize it correctly, resulting in errors.

# Minimal working example

1. When I follow a wiki link containing only English characters, everything works correctly..

`* [Wiki Link only English](wiki-link-only-english.md)`

2. However, when I follow a wiki link that includes Chinese characters, an error occurs.

`* [Wiki Link include Chinese 中文字符](wiki-link-include-chinese-中文字符.md)`

```
Error detected while processing function wiki#link#follow[27]..function wiki#link#follow[23]..wiki#url#follow[12]..wiki#url#handlers#md[1]..wiki#url#handlers#wiki[3]..wiki#url#utils#go_to_file:
line   10:
E480: No match: D:/WeirdData/wiki/wiki-link-include-chinese-<bf>\?<bf>\?<bf>\?<bf>\?.md
```

![wikilink-chinese](https://github.com/user-attachments/assets/1107f55c-985e-4e38-972d-acf6ca86b989)

# Additional note

This issue arose only after I updated wiki.vim to Commit 8cc3f58 yesterday. Before that, I was using Commit b407abe, which operated smoothly. Thus, I suspect this problem may have originated after Commit b407abe.

![wikilink-chinese-commit-b407abe](https://github.com/user-attachments/assets/09798527-709f-442d-a794-51cf4b824dd1)

**Comment by lervag** [comment]
Sorry, I believe this was a regression bug and I think it is fixed now. Please update and check.

Also, I think this is a duplicate of #395.

**Comment by lervag** [comment]
And I'm very sorry for the inconvenience! Thanks for opening the issue and for your patience in me fixing it!

**Comment by VimWei** [comment]
Thanks! It's fixed and working now.

**Comment by lervag** [comment]
Great, glad to hear it works now!
